spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -44,15 +44,14 @@
             "subtype": "Account Book"
           },
           "creators": [{
-            "name": "#{user.formal_name}",
-            "type": "personal",
-            "role": "",
-            "given_name": "#{user.display_name.split.first}",
-            "family_name": "#{user.display_name.split.last}",
-            "identifiers": {
-              "orcid": "#{user.orcid.split('/').last}"
-            },
-            "affiliations": []
+            "person_or_org": {
+              "type": "personal",
+              "given_name": "#{user.formal_name.split(',').last}",
```

Comment:
Formal name is really the field we should have been using this whole time. Display name is typically the same, but I could see a situation where someone uses a nickname or something similar.